PreprocessingScript_step2.sh

```diff
@@ -73,15 +102,31 @@ do
 
 	# ##-------------Nuisance regressors---------------------------
 	echo "====================$workingdir: Nuisance regression: motions, motion devs, PCA noise, trends, gs/wmcsf/csf===================="
-	# Calculate motion derivative regressors (AFNI)---works for small motion defects
-	1d_tool.py -overwrite -infile ./"$workingdir"/EPI_mc.par -derivative -write ./"$workingdir"/motionEPI.deriv.par
-	# get constant, linear, quad trends
-	NR=$(3dinfo -nv ./"$workingdir"/EPI_topup.nii.gz);
-	1dBport -band 0 0 -nodata ${NR} -quad > ./"$workingdir"/quad_regressionEPI.txt
-	# extract CSF/WMCSF/Global signals---noise
-	fslmeants -i ./"$workingdir"/EPI_topup.nii.gz -o ./"$workingdir"/csfEPI.txt -m ./"$workingdir"/EPI_n4_csf_mask.nii.gz	
-	fslmeants -i ./"$workingdir"/EPI_topup.nii.gz -o ./"$workingdir"/gsEPI.txt -m ./"$workingdir"/EPI_n4_mask.nii.gz
```

Comment:
This is the default option, so we need this no matter what was specified in the nuisance parameters.